test/integration/coronavirus_local_restrictions_test.rb

```diff
@@ -327,4 +334,12 @@ def level_two_area
       "country_name" => "England",
     }
   end
+
+  def then_i_see_the_related_links_for_coronavirus
+    within ".gem-c-related-navigation" do
+      assert page.has_link?("Coronavirus (COVID-19): guidance and support", href: "/coronavirus")
```

Comment:
Can we use a reference to the I18n instead of hardcoding this?

Reply:
I'm trying to figure out how we could do this but it's tricky given how we iterate through related navigation in the yaml. Open to suggestions 🤔

Reply:
you could replace the assertions with a loop;
```
      I18n.t("coronavirus_local_restrictions.related_navigation").each do |link|
        assert page.has_link?(link[:title], href: link[:base_path])
      end
```

Reply:
Brilliant Jon 😄 I've never seen that before!